content/docs/get_started/login_requiring.md

```diff
@@ -0,0 +1,163 @@
+---
+$title: Create a login-requiring AMP page
+$order: 3
+---
+Some user interactions with a page, such as leaving a comment, could be conditioned by a login flow. A login flow can be implemented with AMP using the [amp-access](https://www.ampproject.org/docs/reference/components/amp-access) component combined with the [amp-form](https://www.ampproject.org/docs/reference/components/amp-form) component. 
```

Comment:
Change from passive to active throughout, e.g. "can be implemented" to "You can implement". Will make it sound more actionable.